src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_PAIRS_NUMBER = 10;
+    private static int sizeCounter;
```

Comment:
```suggestion
    private int size;
```

why static? If i created 2 instances of StorageImpl class to store Cats and Dogs do they need to have the same size value?

Reply:
changed to instance level